src/app/component/MenuOverlay/MenuOverlay.component.js

```diff
@@ -86,6 +86,7 @@ export default class MenuOverlay extends PureComponent {
                   mix={ { block: 'MenuOverlay', elem: 'Image', mods: itemMods } }
                   src={ icon && media(icon) }
                   ratio="16x9"
+                  onErrorShow={ false }
```

Comment:
Why not check for image here and return null? If image is not found - it is NEEDED to show not found text - it motivates to remove not existing image.